Bottom: 3847800ec946aced8bae825659b28d142e950382
Top:    4d99055bd616e40e92bb546f39e2f83413414c90
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-01-06 14:32:49 +0000

Refresh of adjust-step-export-to-2-3mm

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index c90f11c..a9065ab 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -60,7 +60,7 @@
 #endif
 
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 #define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
 
 static GList *object3d_test_objects = NULL;
diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index d765c3c..02e9528 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -45,7 +45,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 
 
 HID step_hid;
